easybuild/easyconfigs/m/MUSCLE/MUSCLE-3.8.1551_fix-mk-hardcoding.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur /dev/shm/MUSCLE/3.8.1551/GCC-8.2.0-2.31.1.orig/Makefile /dev/shm/MUSCLE/3.8.1551/GCC-8.2.0-2.31.1/Makefile
```

Comment:
@omula Please add a short description + author or upstream reference on top (& adjust the checksum accordingly)

Reply:
Done :)